ext/RMagick/rmimage.c

```diff
@@ -15525,6 +15525,23 @@ static void call_trace_proc(Image *image, const char *which)
 }
 
 
+static VALUE
+rm_trace_creation_body(VALUE img)
+{
+    Image *image = (Image *)img;
+    call_trace_proc(image, "c");
+    return Qnil;
+}
+
+static VALUE
+rm_trace_creation_handle_exception(VALUE img, VALUE exc)
+{
+    Image *image = (Image *)img;
+    DestroyImage(image);
+    rb_exc_raise(exc);
+    return Qnil; /* no reachable */
```

Comment:
I think the comment here should read `/* not reachable */`.

Reply:
Ah, sorry. It is my poor English.